File: src/com/example/Recipe_App/MainActivity.scala
package com.example.Recipe_App

import android.view.View
import android.view.View.OnClickListener
import android.widget.{Toast, Button}
import org.scaloid.common._

/**
 * Created by jacobpreston on 9/14/14.
 */
class MainActivity extends BaseActivity {
  onCreate {
    setContentView(R.layout.main)
    val alex_button = findViewById(R.id.alex_button).asInstanceOf[Button]
    alex_button.setOnClickListener(new OnClickListener {
      override def onClick(v: View) {
        startActivity[SecondActivity]
      }
    })
  }
}

File: src/com/example/Recipe_App/MainActivity.java
package com.example.Recipe_App;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

/**
 * Created by jacobpreston on 10/2/14.
 */
public class MainActivity extends BaseJavaActivity {
    private Button alex_button;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);
        alex_button = (Button)findViewById(R.id.alex_button);
        alex_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, SecondActivity.class);
                startActivity(intent);
            }
        });

    }
}
